config/routes.rb

```diff
@@ -1,4 +1,14 @@
+class AuthConstraint
+  def matches?(request)
+    request.session[:user]["email"].present?
```

Comment:
maybe we should make sure that the email in the session matches a user in the db?